modules/packages/TOML.chpl

```diff
@@ -370,6 +371,16 @@ module TomlParser {
           Date = d;
           return Date;
         }
+        // Time
+        else if ti.match(val) {
+          var raw = getToken(source).split(":");
+          var t = new time(raw[1]: int,
+                           raw[2]: int,
+                           raw[3]: int);
```

Comment:
Won't this be problematic for seconds with a decimal?

Reply:
:+1: Yeah.. I've added fractional seconds support.